src/content/javascript-instrument-page-scope.ts

```diff
@@ -406,7 +419,17 @@ export const pageScript = function() {
     return typeof property === "object";
   }
 
-  function instrumentObject(object, objectName, logSettings = {}) {
+  interface LogSettings {
```

Comment:
Does this actually get injected into the page? I'm not familiar with `interface` so I'm guessing it's typescript-specific. I ask because the code here is stringified and injected into the page. Maybe this gets removed in the build step?

Reply:
Yes, indeed, it gets removed in the typescript compilation step